cypress/e2e/47_adminFIlterSortBy.cy.ts

```diff
@@ -0,0 +1,48 @@
+describe('Super Admin Bounty Filter SortBy', () => {
+  let activeUser = 'alice';
+
+  const bounty: Cypress.Bounty = {
+    title: 'MirzaIssue',
+    category: 'Web development',
+    description: 'This is available',
+    amount: '123',
+    deliverables: 'We are good to go man'
+  };
+
+  beforeEach(() => {
+    cy.login(activeUser);
+    cy.wait(1000);
+  });
+
+  it('Should create six bounties, verify order, change sort, verify new order, and logout', () => {
+    for (let i = 1; i <= 6; i++) {
+      const updatedBounty = { ...bounty, title: `MirzaIssue${i}` };
+      cy.create_bounty(updatedBounty);
+      cy.wait(1000);
+    }
+
+    cy.visit('http://localhost:3007/admin');
+    cy.wait(3000);
+
+    // Assert that the bounties are on the bounties list order in Descending order
+    for (let i = 6; i >= 1; i--) {
+      cy.contains(`MirzaIssue${i}`, { timeout: 10000 }).should('exist');
+    }
+
+    cy.get('[data-testid="Sort_By"]').contains('Sort By:').click();
+    cy.contains('Oldest').click();
+    cy.wait(2000);
+
+    cy.get('.sc-plXmT.fMXCJd').within(() => {
```

Comment:
@MirzaHanan It breaks for me at this point, which button are you trying to click here?

Reply:
Hi @elraphty,
Last button of pagination.

Reply:
> Hi @elraphty, Last button of pagination.

Oh! why do you have to click the last button of the pagination, pass a test id to it if you need to click it

Reply:
Since many bounties are present on the admin side, my oldest bounties appear on the last page because previous old bounties are presented on the first pages.

Reply:
> > Hi @elraphty, Last button of pagination.
> 
> Oh! why do you have to click the last button of the pagination, pass a test id to it if you need to click it

Ok i try with id

Reply:
> Since many bounties are present on the admin side, my oldest bounties appear on the last page because previous 7 days old bounties are presented on the first pages. Then show first.

Oh! Okay

Reply:
Hi @elraphty,
I believe displaying just the first bounty in ascending order is sufficient because sometimes all bounties are on the last page, and at other times, some bounties are on both previous pages and the last page. For example, the first two bounties might be on page 3, and others on page 4.